regparser/tree/xml_parser/preprocessors.py

```diff
@@ -88,6 +89,20 @@ def transform(self, xml):
                     em.tail = ")" + _str(em.tail)
 
 
+class MoveAdjoiningChars(PreProcessorBase):
+    ORPHAN_REGEX = re.compile(ur"(\.|—)")
+
+    def transform(self, xml):
+        # if an e tag has an emdash or period after it, put the
+        # char inside the e tag
+        for e in xml.xpath("//P/E"):
+            orphan = self.ORPHAN_REGEX.match(e.tail)
```

Comment:
@theresaanna I don't believe this has come up in our XML, but hypothetically, `e.tail` could be `None`, in which case I _think_ this would explode.

Would you mind following up with a test that verifies `<E T="03">Things</E>` (i.e. having no "tail"). A simple fix would be `self.ORPHAN_REGEX.match(e.tail or '')` if the `None` is a problem.